### ISSUE TITLE
Doesn't work for all file types

### ISSUE THREAD

**Issue by xpe** [cjwit/selection-word-count#5]
> Selection Word Count
>
> This simple VSCode extension calculates and displays the word count of a document and, when there is a selection, the word count of a selection. It displays both in the status bar.

It worked for a Markdown file, but not in many other file types I tried. Is this expected behavior?

It would be helpful if the documentation reflected the intended behavior of this extension.

**Comment by cjwit** [comment]
Yes, this is the expected behavior and I have updated the README to reflect it. But I am open to a PR, I bet it would be a fairly straightforward one. I just built it for myself and use MD to write. Thanks for pointing out the confusion.

**Comment by cjwit** [comment]
Didn't mean to close this, I'm going to keep it open as a potential enhancement in case someone comes through and has the time to take a look.
